Lib9c/Policy/BlockPolicySource.cs

```diff
@@ -84,15 +84,16 @@ public partial class BlockPolicySource
 
         public const long V100290ObsoleteIndex = 4_913_153;
 
-        // NOTE:
-        // Current block index: 4_945_493
-        // Current time(KST): 2022. 9. 13 오후 6:00:55
-        // Target block index: 5_150_000
-        // Target time(KST): 2022. 10. 12 오전 3:42:19
         public const long V100300ObsoleteIndex = 5_150_000;
 
         public const long V100310ObsoleteIndex = 5_200_400;
 
+        // NOTE:
+        // Current time(KST): 2022. 10. 28 16:26:59, 5_276_632
+        // Target release time(KST): 2022. 11. 09, 5_363_032
+        // Target obsolete time(KST): 2022. 12. 08, 5_579_032
+        public const long V100320ObsoleteIndex = 5_579_032;
```

Comment:
this index is not match any season start index. is this correct?

Reply:
Is it better to unify with the season start index?

Reply:
I reflected it. thank you!